documentation/user_docs/docs/user_guide/Getting_started.rst

```diff
@@ -482,85 +655,201 @@ This should give a plot that looks like this:
+- central position x-coordinate
+- central y-coordinate
+- background
+
+.. note::
+
+   In general the input to a function called by ``func_eval`` can take any form
+   (e.g. a cell array, a structure array, a string, etc.), although if you wish
+   to pass anything other than a vector of parameters, such as that shown above,
+   then it must be packed into a cell array.
 
 .. image:: ../images/Screenshot_SimCut.png
    :width: 300px
    :alt: 2d simulation
 
 
-``func_eval`` works for both sqw and dnd objects with almost the same syntax. For sqw objects pixel information is
-simulated according to the intensity calculated for the data grid, whereas for dnd objects this is not required. It is
-also possible to simulate a dnd from a template sqw object by using an additional keyword argument of the form
+.. note::
+
+   ``func_eval`` works for both ``sqw`` and ``dnd`` objects with almost the same
+   syntax.
+
+
+.. note::
+
+   For ``sqw`` objects, pixel information is simulated according to the
+   calculated intensity for the data grid, whereas for ``dnd`` objects this is
+   not required.
+
+It is also possible to simulate a ``dnd`` from a template ``sqw``
+object by using an additional keyword argument ``'all'`` as follows:
 
 ::
 
-   dnd_sim= func_eval (w_template,@demo_4gauss,[6 1 1 0.1 1.25 6 1],'all');
+   dnd_sim = func_eval(w_template, @demo_4gauss, [6 1 1 0.1 1.25 6 1], 'all');
+
+
+Furthermore one can use the ``'all'`` keyword argument with a template ``dnd``
+object so that intensity is simulated over the entire data range, rather than
+just at the points where there are data in the template object.
+
 
+3D Heisenberg Ferromagnet
+-------------------------
 
-Furthermore one can use the same keyword argument on a template dnd object so that intensity is simulated over the
-entire data range, rather than just at the points where there are data in the template object.
+In this case we will be fitting a full model of S(**Q**,E), using ``sqw_eval``.
 
-There is another way of performing a simulation, using a different method and a simulation function with a slightly
-different input structure. In this case you are fitting a full model of S(**Q**,E), so the function we will demonstrate
-here is a model appropriate for spin excitations of a 3D Heisenberg ferromagnet. The function is called
-``FM_spinwaves_2dSlice_sqw``, and it takes as its inputs arrays (or scalars) for all 3 components of **Q** plus energy,
-as well as the other function parameters (exchange constant etc.). The format of the inputs for this function are thus
-different from those of ``demo_4gauss`` - to see the differences it is probably easiest to examine the code for the two
-functions side-by-side.
+.. note::
+
+   The difference between ``sqw_eval`` and ``func_eval`` is the way in which the
+   arguments are passed to the target function. ``sqw_eval`` passes the
+   positions (h,k,l,e) for each point, while ``func_eval`` simply passes the
+   calculated intensity.
```

Comment:
I thought that calculated intensity was an output from the simulation. This sounds as if it is being passed in